canonical-web-and-design/ubuntu.com
# Re-model packages

## Done

- Revert to previous packages tables and package status release tables

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8001/
- load data with `dotrun exec python3 webapp/security/fixtures/load_sample_cve.py`
- Search now by package e.g. http://0.0.0.0:8001/security/cve?package=gitlab (gitlab is the only package in the dummy data)
